docker-compose.yml

```diff
@@ -146,8 +146,8 @@ services:
       ADDITIONAL_APPS_PATH:
       NEXTCLOUD_TRUSTED_DOMAINS:
     volumes:
-      - /home/jus/repos/nextcloud/stable17:/var/www/html
-      - /home/jus/repos/nextcloud/stable17/apps-extra:/var/www/html/apps-extra
+      - '${STABLE_ROOT_PATH:-/home/jus/repos/nextcloud}/stable17:/var/www/html'
+      - '${STABLE_ROOT_PATH:-/home/jus/repos/nextcloud}/stable17/apps-extra:/var/www/html/app-extra'
```

Comment:
I assume the apps-extra to app-extra is an unrelated change. I'd prefer to keep that at the old path, smae for the others

Reply:
Ah, my bad - that should have been apps.